sdks/python/apache_beam/transforms/core.py

```diff
@@ -1172,7 +1172,7 @@ class Windowing(object):
 
   def __init__(self, windowfn, triggerfn=None, accumulation_mode=None,
                output_time_fn=None):
-    global AccumulationMode, DefaultTrigger
```

Comment:
Do you want to move this change to a non-pylint change?

@robertwb do you know why we had these defined here?